app/src/main/res/layout/item_speaker.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="?android:attr/listPreferredItemHeight"
+  android:background="?selectableItemBackground"
+  android:orientation="horizontal">
+
+  <ImageView
+    android:id="@+id/speaker_image"
+    android:layout_width="wrap_content"
```

Comment:
This will likely be made fixed (I will edit this layout as part of #51 as I'll use these there too)